esp/logging/loggingmanager/CMakeLists.txt

```diff
@@ -52,7 +52,7 @@ include_directories (
     ./..
 )
 
-ADD_DEFINITIONS( -D_USRDLL -DLOGGINGCOMMON_EXPORTS -DLOGGINGMANAGER_EXPORTS )
+ADD_DEFINITIONS( -D_USRDLL -DLOGGINGCOMMON_EXPORTS -DLOGGINGMANAGER_EXPORTS -DWSLOGGING_API_LOCAL )
```

Comment:
loggingmanager links go logginglib which incldues ws_loggingservice_esp.cpp (and is now exporting the symbols),
I think     ${ESPSCM_GENERATED_DIR}/ws_loggingservice_esp.cpp can now be removed from loggingmanager (and this -DWSLOGGING_API_LOCAL)

Reply:
@jakesmith These various logging dependencies need weeding through.  In the case you mention loggingmanager seems to be using the constructor directly, rather than the factory.  And I think some of the logging stuff in general may actually do things like derive new classes from the generated ones (not sure where just needs to be worked out).  Anyway it should definitely all be cleaned up, but not in this PR... which Gavin just merged.

Reply:
fair enough, main concern was if the exported symbols from logginglib, and the local syms included in same module within loggingmanager would clash in some way, but build was clean.